### ISSUE TITLE
High Bandwidth Usage on Dedicated Server

### ISSUE THREAD

**Issue by frogwallet** [RootsTeam/Embers#52]
Hi Team,

We added Embers to my server today and as players begun using the mod we saw bandwidth usage for all users increase significantly.

Each user was using 400 KB/s of bandwidth (we should only be seeing 20-30KB/s) and dropped back to regular levels immediately after removing all embers blocks. 

We tested with item/fluid pumps and the Ember Bore.

Currently using Embers 0.101.jar

**Comment by elucent** [comment]
Hmmm. Any profiling tools you have that can get me more specific information? I've tested Embers on a couple servers, and even in a vanilla setting Embers pipes and machines took up less tick time than the randomly spawned vanilla mobs in the chunk.

**Comment by frogwallet** [comment]
We didn't test by tick time as we didn't notice any issues with TPS - My players complained of lag and one of them mentioned that they were downloading near the limit of their ADSL connection just playing the server. This prompted me to look at the bandwidth use of the server, which was astronomical. The only thing that had changed was the addition of the Embers mod to our server.

I looked at the upload of the server which was sitting at 4MB/s consistently. I confirmed one of our players had started using the embers mod. We removed his machines and with every machine removed, bandwidth usage decreased, eventually returning to normal levels when all machines had been removed.
